Source/AliveLibAE/Abe.cpp

```diff
@@ -1386,9 +1386,9 @@ void Abe::Update_449DC0()
 
         if (field_114_flags.Get(Flags_114::e114_Bit8_bInvisible))
         {
-            if (!field_170_invisible_timer)
+            if (!field_170_invisible_timer.mTimer)
             {
-                field_170_invisible_timer = sGnFrame_5C1B84 + 2;
+                field_170_invisible_timer.MakeTimer(2);
```

Comment:
I would prob rename MakeTimer to be Start(2) as you are starting a 2 frame timer, or AfterFrames(2) or something